_posts/2020-01-07-aksengine-advanced-networking-part1.md

```diff
@@ -133,14 +124,18 @@ The API model file provides various configurations which aks-engine uses to crea
 You should only provide `keyData`, `clientId` and `secret` fields in `aks.json` to be able to run the **build** script.
 
 ## Build Script
-To build and deploy the kubernetes cluster on a custom Azure VNET, we'll use `build.sh` script. The script will:
+To build and deploy the kubernetes cluster on a custom Azure VNET, we'll use [`build.sh`](/blog/assets/aksengine-advanced-networking/build.sh) script. The script will:
```

Comment:
I'd suggest to remove code quotes from build.sh (and similar links to scripts in the whole file) , so that the fact that it's a link would be clearer to the reader.

Reply:
Yeah. Good catch. I removed quotes from linked texts.
Thank you.